azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/screens/ChatScreen.kt

```diff
@@ -72,11 +76,11 @@ internal fun ChatScreen(
                     scrollState = listState,
                 )
             }
-
-            // TypingIndicatorView(viewModel.typingParticipants.toList())
         },
         bottomBar = {
             Column() {
+                TypingIndicatorView(viewModel.typingParticipants.toList())
+                Spacer(modifier = Modifier.height(20.dp))
```

Comment:
can this be padding inside TypingIndicatorView via a modifier.

When the Indicator isn't visible, we probably don't want the padding?